tools/deviceGen/input/device_features.json

```diff
@@ -0,0 +1,2666 @@
+            "value": false
+        },
+        {
+            "type": "bool",
+            "name": "fragmentShaderPixelInterlock",
+            "value": false
+        },
+        {
+            "type": "bool",
+            "name": "fragmentShaderShadingRateInterlock",
+            "value": false
+        },
+        {
+            "groupComment": [
+                "[DO NOT EXPOSE] Expose nothing to do with video atm",
+                "YcbcrImageArraysFeaturesEXT",
+                "VK_EXT_ycbcr_image_arrays",
+                "[DEPRECATED] Vulkan 1.2 Core non-optional",
+                "VK_KHR_uniform_buffer_standard_layout",
+                "[DO NOT EXPOSE] provokingVertexLast will not expose (we always use First Vertex Vulkan-like convention), anything to do with XForm-feedback we don't expose",
+                "ProvokingVertexFeaturesEXT",
+                "VK_EXT_provoking_vertex",
+                "[TODO LATER] Requires API changes",
+                "VK_EXT_full_screen_exclusive",
+                "[DO NOT EXPOSE] We don't support yet",
+                "VK_EXT_headless_surface",
+                "[DEPRECATED] Core in Vulkan 1.3 and NAbla Core Profile",
+                "VK_KHR_buffer_device_address",
+                "[DO NOT EXPOSE] We don't support yet",
+                "VK_EXT_extension_259",
+                "[TODO] need impl, this feature introduces new/more pipeline state with VkPipelineRasterizationLineStateCreateInfoEXT",
+                "LineRasterizationFeaturesEXT",
+                "VK_EXT_line_rasterization"
+            ]
+        },
+        {
+            "type": "bool",
+            "name": "rectangularLines",
+            "value": false
+        },
+        {
+            "type": "bool",
+            "name": "bresenhamLines",
+            "value": false
+        },
+        {
+            "type": "bool",
+            "name": "smoothLines",
+            "value": false
+        },
+        {
+            "type": "bool",
+            "name": "stippledRectangularLines",
+            "value": false
+        },
+        {
+            "type": "bool",
+            "name": "stippledBresenhamLines",
+            "value": false
+        },
+        {
+            "type": "bool",
+            "name": "stippledSmoothLines",
+            "value": false
+        },
+        {
+            "groupComment": [
+                "[NAbla core Profile LIMIT] ",
+                "ShaderAtomicFloatFeaturesEXT",
+                "VK_EXT_shader_atomic_float",
+                "[DEPRECATED] MOVED TO Vulkan 1.2 Core",
+                "HostQueryResetFeatures",
+                "VK_EXT_host_query_reset",
+                "[DO NOT EXPOSE] just reserved numbers, extension never shipped",
+                "VK_GGP_extension_263",
+                "VK_BRCM_extension_264",
+                "VK_BRCM_extension_265",
+                "[TODO] need impl",
+                "IndexTypeUint8FeaturesEXT",
+                "VK_EXT_index_type_uint8"
```

Comment:
move comment down, also I think there's a `uint8_t` index feature above (I've just read).

So maybe don't expose and say that EXT has been incorporated into 1.3 core or wherever that feature is?